projects/js-packages/ai-client/src/chrome-ai/get-availability.ts

```diff
@@ -31,6 +32,45 @@ function getAiAssistantFeature() {
 	return getFeature();
 }
 
+/**
+ * Fetch an experiment assignment.
+ *
+ * @param {boolean} asConnectedUser - Whether the user is connected.
+ * @return {Function} A function that fetches an experiment assignment.
+ */
+const fetchExperimentAssignment =
```

Comment:
How come we're [duplicating](https://github.com/Automattic/jetpack/blob/4961a2996f8e90b2dfefcd006f391c432daa7306/projects/packages/explat/src/client/assignment.ts#L7) this function in `ai-client/src/chrome-ai/get-availability.ts` instead of adjusting it in the jetpack-explat package? Testing purposes and then cleanup later?